homeassistant/components/smartthings/sensor.py

```diff
@@ -220,6 +220,15 @@
     Capability.oven_setpoint: [
         Map(Attribute.oven_setpoint, "Oven Set Point", None, None, None)
     ],
+    Capability.power_consumption_report: [
+        Map(
+            Attribute.power_consumption,
+            "Energy Consumption",
+            ENERGY_KILO_WATT_HOUR,
+            DEVICE_CLASS_ENERGY,
+            STATE_CLASS_MEASUREMENT,
```

Comment:
PowerConsumptionReport is historical and is not current measurement. It should not be classified as `STATE_CLASS_MEASUREMENT`

Reply:
shoot I must have glossed over the start and end 😞

Reply:
I mean it still increments over time, and from what I understand is the energy dashboard would take the data from 00:00 to now to display it on the dash, even with a delay wouldn't this still be viable?

Reply:
> the energy dashboard would take the data from 00:00 to now

Yes, but what I'm saying is the `PowerConsumptionReport` capability is a report of utilization _in the past_ (from `start` to `end`), _not_ a current measurement.

Per the HA Sensor Entity docs:

> Examples of what should be classified measurement are: current temperature, accumulated energy consumption, accumulated cost. Exampled of what should not be classified as measurement: Forecasted temperature for tomorrow, yesterday's energy consumption or anything else that doesn't include the current measurement.

This capability should not be classified as measurement because it dones't include current AND is in the past.

Reply:
If end = now, it is measurement, because the sum includes the current value. If end < now, it is not a measurement. We currently don't have the option to import historic data.

Reply:
@balloob here's some sample data from one of my devices:
```json
{
    "powerConsumption": {
        "value": {
            "energy": 1412400,
            "deltaEnergy": 30,
            "power": 117,
            "powerEnergy": 21.55886583526929,
            "persistedEnergy": 0,
            "energySaved": 0,
            "start": "2021-07-30T20:45:52Z",
            "end": "2021-07-30T21:00:44Z"
        },
        "timestamp": "2021-07-30T21:00:44.099Z"
    }
}
```
The `end` = `now`.